.circleci/config.yml

```diff
@@ -33,11 +33,27 @@ jobs:
       - run:
           name: Lint ERBs
           command: 'bundle exec erblint --lint-all --format compact'
+  accessibility-features:
```

Comment:
Ticket adding this to orb?

Reply:
I would like to defer until we have this in 2-3 codebases, which gives us time to work out the best patterns.